ChangeLog.md

```diff
@@ -1,3 +1,17 @@
-# Changelog for ide
+## 0.1
```

Comment:
This should not be on the same level as the rest of the headlines. the others should have three '#' imo.

Also, I would like to keep the top-level headline `# Changelog`. I think it is a nicer overview.